### ISSUE TITLE
Window blinks on World Quest items and Auctions

### ISSUE THREAD

**Issue by Beet4** [kemayo/wow-appearancetooltip#15]
AppearanceTooltip window blinks when hovering over items in World Quests and Auctions in the auction house. See gif below. The item does not seem to be shown either.

![Wow 2022-12-10 00 36 49](https://user-images.githubusercontent.com/38279393/206812344-ba49e94c-6a1e-4244-b9fd-9ce6f91168eb.gif)

**Comment by Beet4** [comment]
I've uploaded a quick video on YouTube which shows the bug better. Here it is: https://www.youtube.com/watch?v=vNbmh_h4RqU

**Comment by Beet4** [comment]
Any update on this issue? It is still present as per my video above. :)

**Comment by kemayo** [comment]
I think this is related to certain camera angles -- could you say whether you've got the "zoom on [worn/held] items" settings disabled?

**Comment by Beet4** [comment]
I'm on vacation right now so can't check but I am certain that setting is disabled yes. I have it so I see my entire character naked except for wearing the item in question.

**Comment by Beet4** [comment]
Sorry for late reply. Yes, those options are disabled:
![image](https://github.com/kemayo/wow-appearancetooltip/assets/38279393/1c228629-3cda-44b0-8a92-7f9abc61447c)

![Wow 2023-11-15 23 11 43](https://github.com/kemayo/wow-appearancetooltip/assets/38279393/c25c3da8-d12a-44f4-b276-762992df786f)
![Wow 2023-11-15 23 12 05](https://github.com/kemayo/wow-appearancetooltip/assets/38279393/f6f68a2b-f52b-4ac3-ab62-bc7fd8809620)

Also note that the item is not showing, it shows for a split second, then after the first blink it disappears. 

Enabling zoom on worn/held item make the window stop blinking.

**Comment by Faulkal** [comment]
will this be fixed?

**Comment by Beet4** [comment]
Any updates on this issue? :)

**Comment by kemayo** [comment]
Okay, I think I have worked this one out. It looks like it was an issue with some code that was trying to address a bug with the DressUp functionality that existed back when I first wrote this, where if I didn't reset the model display immediately when displaying the full character model then the model would display off-center. Having tested just now, it looks like (a) that bug isn't there any more, and (b) since that workaround was added the events fired by resetting the model changed in such a way that it'd hide-then-show the model, triggering an endless loop of the model getting redrawn.

I'm not entirely sure why this is only exposed when the tooltip is specifically attached to a world quest, admittedly.
